packages/forms/src/templates/ArrayFieldTemplate.js

```diff
@@ -8,46 +8,42 @@ import IconsProvider from '@talend/react-components/lib/IconsProvider';
 import theme from './ArrayFieldTemplate.scss';
 
 function FieldTemplate({ element, cantDelete }) {
-	const arrayElement = classNames(theme.arrayElement, element.itemData.isClosed && theme.closed);
+	const elementClasses = classNames(theme.element, element.itemData.isClosed && theme.closed);
 	return (
-		<div className={arrayElement}>
-			{
-				<div className={theme.control}>
-					<button
-						type="button"
-						name={`btn-delete-element-${element.index}`}
-						disabled={cantDelete}
-						className={theme.delete}
-						onClick={element.onDropIndexClick(element.index)}
-						title="Delete"
-					>
-						<Icon name="talend-trash" />
-					</button>
-					{!element.itemData.isClosed && (
-						<div className={theme.orderaction}>
-							<button
-								type="button"
-								name={`btn-move-element-up-${element.index}`}
-								disabled={!element.hasMoveUp}
-								onClick={element.onReorderClick(element.index, element.index - 1)}
-								title="Move Up"
-							>
-								<Icon name="talend-caret-down" transform="flip-vertical" />
-							</button>
-							<button
-								type="button"
-								name={`btn-move-element-down-${element.index}`}
-								disabled={!element.hasMoveDown}
-								onClick={element.onReorderClick(element.index, element.index + 1)}
-								title="Move Down"
-							>
-								<Icon name="talend-caret-down" />
-							</button>
-						</div>
-					)}
-				</div>
-			}
-			<div className={theme.element}>{element.children}</div>
+		<div className={theme.arrayElement}>
+			<div className={theme.control}>
+				<button
```

Comment:
like said on top, can you use <Action ...> ?

Reply:
like i said to @jmfrancois its the legacy Form API and i don't really wish to spend more time than necessary since we are going to move onto the new Form API